tests/automotive-IMA/basicmath_large/Makefile

```diff
@@ -0,0 +1,18 @@
+CC		:=	riscv64-unknown-elf-gcc
+OBJDUMP := riscv64-unknown-elf-objdump --disassemble-all --disassemble-zeroes --section=.text --section=.text.startup --section=.data
+
+TARGET	:= basicmath_large.c rad2deg.c cubic.c isqrt.c
+GCC_OPTS = -m32 -Wa,-march=RV32IMA -std=gnu99 -msoft-float
```

Comment:
Yes I was able to build the executable using this makefile.

I just copied the basicmath_small folder in the same folder where liba_sysc and rv_hal folders were present. After running make command for libac_sysc. I entered the basicmath_small folder and ran make command. It generated the required elf file.